models/GroupInvites.js

```diff
@@ -0,0 +1,35 @@
+'use Strict';
+
+module.exports = (sequelize, DataTypes) => {
+    const GroupInvites = sequelize.define('GroupInvites', {
+
+        //auto generate UUID format - can be UUIDV1 or UUIDV4
+        uuid: {
+            type: DataTypes.UUID,
+            primaryKey: true,
```

Comment:
Not sure if we can have `uuid` without `primaryKey`, but if we can, let's remove it.

Also, I guess we can have `unique: true` here as well

Reply:
I will leave this in bc having primaryKey:true disables the default _id to generate.